### ISSUE TITLE
Operations: Merge NREL 2.6.0 into NRCan Branch

### ISSUE THREAD

**Issue by phylroy** [canmet-energy/btap_tasks#149]
### Description
Merge NREL latests standards into nrcan branch. 

### Approach
1) Merge all changes from master into nrcan_nrel_merge
2) Merge all changes from nrcan into nrcan_nrel_merge
3) Update BTAP_DE to use 2.6.0
4) Merge nrcan_nrel_merge into nrcan. 
Ensure the circle ci now uses  

### Testing Plan
Ensure circleci passes and all applications work on BTAP-DE2.6

### Waiting On
If this task is dependent on another task or NREL. Include notes Here. 

### Repositories Involved

https://github.com/canmet-energy/btap-development-environment
https://github.com/canmet-energy/docker-openstudio
https://github.com/canmet-energy/btap
https://github.com/NREL/openstudio-standards/tree/nrcan

### Select the Project Code you will be charging to 2018-2019

| Client 	|WSBE  	|Name   	|Manager   	
|----------	|---	|---	|---	
|PERD   |P-002518.001.03	|Building Archetype	|Phylroy Lopez
